JacobF94/NSS-BackEnd-Capstone-CookBook
# user can edit their recipes

#### Changes Made
1. Added a new form for editing pre existing recipes
2. users can edit from their profile page
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout jf-editRecipes
    ```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
4. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does BLANK, then BLANK should happen.
5. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
